src/extensions/default/HealthData/HealthDataManager.js

```diff
@@ -130,9 +130,31 @@ define(function (require, exports, module) {
     }
 
     // Get Analytics data
-    function getAnalyticsData() {
+    function getAnalyticsData(EventParams) {
```

Comment:
We always name Modules/Custom Objects(Contructed ones) with **TitleCase**, and all variables and functions using **camelCase**

So the parameter in the function definition should be **eventParams** not "EventParams"